ckanext/scheming/validation.py

```diff
@@ -83,6 +88,27 @@ def validator(key, data, errors, context):
     return validator
 
 
+@scheming_validator
+def scheming_isodatetime(value, context):
+    def validator(key, data, errors, context):
+        value = data[key]
+        if type(value) == list:
+            value = data[key] = value[0] + ' ' + value[1]
```

Comment:
This will result in a server error if the list has fewer than two elements or if the types in the list aren't unicode or str. You can wrap it in a try/except to catch those cases. Also, `isinstance(value, list)` is a little better than using `type(value) == list`